collections/_article/release-candidate-godot-4-2-rc-2.md

```diff
@@ -0,0 +1,101 @@
+- Rendering: Prevent crash in `_nvapi_disable_threaded_optimization` when attached to renderdoc ([GH-85121](https://github.com/godotengine/godot/pull/85121)).
+- Rendering: Avoid division by zero in the fix surface compatibility routine ([GH-85138](https://github.com/godotengine/godot/pull/85138)).
+- Rendering: Fix potential double-close of draw command label ([GH-85147](https://github.com/godotengine/godot/pull/85147)).
+- Rendering: Enable non-multiview advanced shader group whenever advanced shaders are requested ([GH-85194](https://github.com/godotengine/godot/pull/85194)).
+- Shaders: Make `AMOUNT_RATIO` constant in the shader language specification ([GH-85086](https://github.com/godotengine/godot/pull/85086)).
+- Shaders: Set some dialogs in the VisualShader editor to be exclusive ([GH-85205](https://github.com/godotengine/godot/pull/85205)).
+- Documentation and translation updates.
+
+This release is built from commit [`1ba920fad`](https://github.com/godotengine/godot/commit/1ba920fada9efc8c4476ded50fe673b8db541366).
+
+## Downloads
+
+{% include articles/download_card.html version="4.2" release="rc2" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- .NET build requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0), or [8.0](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) installed in a standard location.
+- To export to Android, .NET 7.0 or later is required. To export to iOS, .NET 8.0 is required. Make sure to set the target framework in the `.csproj` file.
+
+{% include articles/prerelease_notice.html %}
+
+## Known issues
+
+We are aware of some regressions which still affect this release candidate, but which we may not be able to solve in time for the stable release. They're still tracked as high priority and should be fixed soon after the release with backports to 4.2.x maintenance releases.
+
+- TileMap editing performance on macOS got significantly worse when using the ANGLE backend for the Compatibility renderer ([GH-84591](https://github.com/godotengine/godot/issues/84591)). We have good hunches for what causes the issue but it will take some time to solve properly. In the meantime, you can change the `rendering/gl_compatibility/driver.macos` project setting to `opengl3` instead of the new default `opengl3_angle` to restore the 4.1 behavior if you are affected by this regression.
+- Capture track in animations no longer work after the unification of the AnimationPlayer and AnimationTree processing in AnimationMixer ([GH-83166](https://github.com/godotengine/godot/issues/83166)). This feature was specific to AnimationPlayer and was lost in the refactoring. We're evaluating what's needed to restore this feature.
```

Comment:
I'm not sure if this should be "Capture tracks in animations no longer work" or "Capture track in animations no longer works", maybe?

Reply:
The former makes sense.